frontend/src/components/common/ReleaseNotes/ReleaseNotes.tsx

```diff
@@ -49,6 +62,17 @@ export default function ReleaseNotes() {
             const storedAppVersion = helpers.getAppVersion();
             let releaseNotes = '';
             if (storedAppVersion && semver.lt(storedAppVersion as string, currentBuildAppVersion)) {
+              // attach a timeout which checks after 10 second of fetching release
```

Comment:
We already use Octokit before this to get the latest release. So if that's slow, I guess we'll be slow to get here.
This means we should likely have also the "fetching" popup being spawned after a timeout before that initial Octokit call.

Reply:
added at both places